content/academy/data_collection_basics/api_scraping.md

```diff
@@ -90,20 +76,39 @@ Since our request doesn't have any body/payload, we just need to analyze the URL
 
 Understanding an API's various configurations helps with creating a game-plan on how to best scrape it, as many of the parameters can be utilized for easy pagination, or easy data-filtering. Additionally, these values can be mapped to a scraper's configuration options, which overall makes the scraper more versatile.
 
-Based on our observations, we can modify the URL and utilize the "`limit`" option to return 200 songs in one request, instead of just 20:
+Let's say we want to receive all of the user's tracks in one request. Based on our observations of the endpoint's different parameters, we can modify the URL and utilize the "`limit`" option to return more than just twenty songs:
 
 ```
-https://api-v2.soundcloud.com/users/141707/tracks?client_id=zdUqm51WRIAByd0lVLntcaWRKzuEIB4X&limit=200
+https://api-v2.soundcloud.com/users/141707/tracks?client_id=zdUqm51WRIAByd0lVLntcaWRKzuEIB4X&limit=99999
 ```
 
+By using the ridiculously large number of `99999`, we ensure that all of the user's tracks will be captured in this single request.
+
```

Comment:
Sorry. I guess I wasn't entirely clear before. My point was: how did we find that we can use the number 200, or 99999, or any other number. I think it might not be entirely obvious to people who are new to API scraping that they can just change it to whatever they want and see what happens. And that they might get overflow errors, in which case they need to reduce the number, and so on.